python/py-cliff/Portfile

```diff
@@ -16,21 +16,25 @@ long_description    cliff is a framework for building command line \
                     provide subcommands, output formatters, and other \
                     extensions.
 homepage            https://docs.openstack.org/cliff/latest/
-checksums           rmd160  6dbe1b37f6643b4ab38a45509e0e4d1db3c45b80 \
-                    sha256  aa8d404aa2d6b4d8639c61bd6dc47acb3656ebc3fc025b1b7bb07af2baef785f \
-                    size    83609
+checksums           rmd160  370f093748ab5b28e5f6776cadde5e23e717826b \
+                    sha256  6ca45f8df519bbc0722c61049de7b7e442a465fa7f3f552b96d735fa26fd5b26 \
+                    size    84250
 
-python.versions     38 39 310 311
+python.versions     38 39 310 311 312
 
 if {${subport} ne ${name}} {
     depends_build-append \
                     port:py${python.version}-pbr
 
+    if {${python.version} < 310} {
+        depends_build-append  port:py${python.version}-importlib-metadata
```

Comment:
this should be "depends_lib"

Reply:
Thanks for checking and suggesting the corrections @reneeotten.